sdk/src/test/java/io/opentdf/platform/sdk/TDFTest.java

```diff
@@ -492,6 +498,32 @@ public void testCreateTDFWithMimeType() throws Exception {
         assertThat(reader.getManifest().payload.mimeType).isEqualTo(mimeType);
     }
 
+    @Test
```

Comment:
Can we assert the version and also that segments and rootSign are hex encoding when version is less than 4.3.0.